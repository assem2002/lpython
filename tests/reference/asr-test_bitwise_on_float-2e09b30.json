{
    "basename": "asr-test_bitwise_on_float-2e09b30",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_bitwise_on_float.py",
    "infile_hash": "50e0d4c18ad2fe0db0aedfaa5b3db792db545aeb8ef09a4cb7c4f193",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_bitwise_on_float-2e09b30.stderr",
    "stderr_hash": "bfd9481140ad60fba6aef69dd04bb1e832e0bafaea1041eaa9b91f84",
    "returncode": 2
}