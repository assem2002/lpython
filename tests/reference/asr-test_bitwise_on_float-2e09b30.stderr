semantic error: Unsupported binary operation on floats: '<<'
 --> tests/errors/test_bitwise_on_float.py:8:11
  |
8 |     print(f1 << f2)
  |           ^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
